src/Servers/IIS/IIS/src/NativeMethods.cs

```diff
@@ -91,9 +92,10 @@ public enum REQUEST_NOTIFICATION_STATUS
     private static partial int http_get_application_properties(out IISConfigurationData iiConfigData);
 
     [LibraryImport(AspNetCoreModuleDll)]
+    [SuppressMessage("LibraryImportGenerator", "SYSLIB1051:Specified type is not supported by source-generated P/Invokes", Justification = "The enum is handled by the runtime.")]
```

Comment:
What's the issue?

Reply:
It doesn't appreciate that the generated type HTTP_REQUEST_PROPERTY is an enum, but it works fine.